packages/astro-ricos/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@wix/astro-ricos",
+  "type": "module",
+  "version": "1.0.0",
+  "exports": {
+    ".": "./index.js"
+  },
+  "scripts": {
+    "dev": "yarn astro dev",
+    "build": "yarn astro build",
```

Comment:
I think you need a typescript build here nope? to have the ts files translated to js? I'm not sure how an astro component that uses ts from node_modules will work in a project?